src/components/Tiles/TopServices.js

```diff
@@ -6,16 +6,17 @@ import ArrowRight from 'components/SVGs/ArrowRight';
 import Tile from './Tile';
 
 const TopServices = ({ tiles, title, url, locale, extraClasses }) => (
-  <div className={`coa-TopServices ${extraClasses}`}>
+  <div className={'coa-TopServices ' + (extraClasses ? extraClasses : '')}>
```

Comment:
wouldn't this work as:
`coa-TopServices ${!!extraClasses && extraClasses}`
or just
`coa-TopServices ${extraClasses ? extraClasses : ''}`

Is there something about [template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) that's causing issues?

Reply:
Updated the template literal to your second snippet. The others were returning false or undefined, so it was just a cleanup. Thanks for spotting!